modules/manually-gathering-logs-without-SSH.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// *installing/installing-gather-logs.adoc
+
+[id="installation-manually-gathering-logs-without-SSH_{context}"]
+= Manually gathering logs without SSH access to your host(s)
+
+If you do not have SSH access to your node, you can access the systems journal
+to investigate what is happening on your host.
+
+.Prerequisites
+
+* Your {product-title} installation must be complete.
+* Your API service is still functional.
+* You have system administrator privileges.
+
+.Procedure
+
+. Access journald unit logs under `/var/log` by running:
```

Comment:
Consider marking up?[1] Or maybe Journal, though I don't see us having done that anywhere in our repo so far.

[1] https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/system_administrators_guide/s1-interaction_of_rsyslog_and_journal